.travis.yml

```diff
@@ -1 +1,4 @@
-language: java
\ No newline at end of file
+language: java
+
+jdk:
```

Comment:
There was some problem with Mockito and jdk 11. I believe this repo should be targeting jdk8 anyway. Let me know if you disagree (in which case I'll take my toys and go).